modules/@apostrophecms/rich-text-widget/index.js

```diff
@@ -106,6 +106,7 @@ module.exports = {
       creditUrl: 1
     },
     minimumDefaultOptions: {
+      marksPreview: true,
```

Comment:
Because it inserts a dev-authored style into a carefully carved out admin UI it just felt like it had the potential to jack up the view, even though I'm trying to clamp down obvious properties with `!important`. 
If the 'safety' of this is outweighed by the 'why would I want to hold this option in my head' I could back it out.

Reply:
Decided to remove this option for now